tests/Integration/ORM/CascadeEntityFactoryRelationshipTest.php

```diff
@@ -27,15 +27,15 @@ final class CascadeEntityFactoryRelationshipTest extends EntityFactoryRelationsh
      */
     public function ensure_to_one_cascade_relations_are_not_pre_persisted(): void
     {
-        $contact = $this->contactFactory()
```

Comment:
sorry for all this noise, I had to migrate from "non static" to "static" for all these "factory" methods, to satisfy PHPUnit 11